app/Http/Controllers/Eportalcontroller.php

```diff
@@ -65,15 +77,16 @@ public function store(Request $request)
             $file->save();
             if($request->abc != null)
             {
-                return redirect('/show?abc='.$file->parent_directory)->with("success","Successfully Uploaded");
+                return redirect('/files'.'/'.$file->parent_directory)->with("success","Successfully Uploaded");
```

Comment:
you can use named routing in Laravel.
https://laravel.com/docs/5.7/routing#named-routes
```suggestion
                return redirect(route('directory', $file->parentDirectory))->with("success","Successfully Uploaded");
```